subprojects/kotlin-dsl-plugins/build.gradle.kts

```diff
@@ -6,7 +6,7 @@ plugins {
 description = "Kotlin DSL Gradle Plugins deployed to the Plugin Portal"
 
 group = "org.gradle.kotlin"
-version = "3.2.3"
+version = "3.2.4"
```

Comment:
Weird that we use `3.2.4` here but have added an entry for `3.2.3` in dependency verification
Is it because the `3.2.3` elsewhere will be updated when updating the wrapper?

Reply:
Here is the "next" version that isn't published yet and only consumed by integration tests.

A subsequent PR will update the wrapper to use `3.2.3`.

The dependency verification metadata is updated in this PR mostly for the Gradleception CI job.